dipy/align/tests/test_vector_fields.py

```diff
@@ -1055,21 +1085,17 @@ def test_compose_vector_fields_3d():
     input_grid2world = gt_affine
     target_grid2world = gt_affine
 
-    disp1, assign1 = vfu.create_random_displacement_3d(np.array(input_shape,
-                                                       dtype=np.int32),
-                                                       input_grid2world,
-                                                       np.array(tgt_sh,
-                                                       dtype=np.int32),
-                                                       target_grid2world)
+    disp1, assign1 = vfu.create_random_displacement_3d(
+        np.array(
+            input_shape, dtype=np.int32), input_grid2world, np.array(
+            tgt_sh, dtype=np.int32), target_grid2world)
     disp1 = np.array(disp1, dtype=floating)
     assign1 = np.array(assign1)
 
-    disp2, assign2 = vfu.create_random_displacement_3d(np.array(input_shape,
-                                                       dtype=np.int32),
-                                                       input_grid2world,
-                                                       np.array(tgt_sh,
-                                                       dtype=np.int32),
-                                                       target_grid2world)
+    disp2, assign2 = vfu.create_random_displacement_3d(
+        np.array(
+            input_shape, dtype=np.int32), input_grid2world, np.array(
+            tgt_sh, dtype=np.int32), target_grid2world)
```

Comment:
I think something like this would be much more readable:
```
    disp2, assign2 = vfu.create_random_displacement_3d(
        np.array(input_shape, dtype=np.int32),
        input_grid2world,
        np.array(tgt_sh, dtype=np.int32),
        target_grid2world)
```
Putting more spaces makes `pep8` complain about over-indenting.